packages/vue-app/template/utils.js

```diff
@@ -261,6 +261,10 @@ export function promisify (fn, context) {
 // Imported from vue-router
 export function getLocation (base, mode) {
   let path = decodeURI(window.location.pathname)
+  // To get matched with sanitized router.base
+  if (!/\/$/.test(path)) {
```

Comment:
We can also use [`endsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith)

Reply:
I'm agree, didn't see this method previously, lol
Fixed